go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/giantswarm/app-operator
+
+go 1.13
+
+require (
+	github.com/giantswarm/apiextensions v0.2.0
+	github.com/giantswarm/appcatalog v0.1.11
+	github.com/giantswarm/apprclient v0.2.0
+	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/e2esetup v0.2.0
+	github.com/giantswarm/errors v0.2.0
+	github.com/giantswarm/exporterkit v0.2.0
+	github.com/giantswarm/helmclient v0.2.0
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/kubeconfig v0.2.0
+	github.com/giantswarm/microendpoint v0.2.0
+	github.com/giantswarm/microerror v0.2.0
+	github.com/giantswarm/microkit v0.2.0
+	github.com/giantswarm/micrologger v0.3.1
+	github.com/giantswarm/operatorkit v0.2.0
+	github.com/giantswarm/versionbundle v0.2.0
+	github.com/google/go-cmp v0.4.0
+	github.com/prometheus/client_golang v1.3.0
+	github.com/spf13/afero v1.2.2
+	github.com/spf13/viper v1.6.2
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.16.6
+	k8s.io/apimachinery v0.16.6
+	k8s.io/client-go v0.16.6
+	k8s.io/helm v2.16.4+incompatible
+)
+
+// Workaround for https://github.com/sirupsen/logrus/issues/570
+// Additional reading: https://github.com/golang/go/issues/28489
+// Solution: https://github.com/moby/moby/issues/39302
+replace github.com/docker/docker => github.com/docker/engine v1.4.2-0.20191113042239-ea84732a7725
```

Comment:
Cool 👍 all this will get simplified once https://github.com/giantswarm/app-operator/pull/223 is merged.